app/lib/utils/getFilteredOutTermsQuery.js

```diff
@@ -1,47 +1,69 @@
 const constants = require('../constants');
 
-function getFilter(searchType) {
-  const mustNotClause = [
-    { match: { name: 'Marie Stopes' } },
-    { match: { name: 'Young People Friendly Practice' } },
-    { match: { name: 'BPAS' } },
-  ];
+const mustNotClause = [
+  { match: { name: 'Marie Stopes' } },
+  { match: { name: 'Young People Friendly Practice' } },
+  { match: { name: 'BPAS' } },
+];
+
+const queryFilters = {};
+
+queryFilters[constants.searchTypes.sexperts] = {
+  must: [
+    { match: { serviceType: 'Sexual health information and support' } },
+    { match: { venueType: 'Clinic' } },
+  ],
+  must_not: mustNotClause,
+};
 
-  if (searchType === constants.searchTypes.sexperts) {
-    return {
-      must: [
-        { match: { serviceType: 'Sexual health information and support' } },
-        { match: { venueType: 'Clinic' } },
-      ],
-      must_not: mustNotClause,
-    };
-  } else if (searchType === constants.searchTypes.kits16to24) {
-    return {
-      minimum_should_match: 1,
-      must_not: mustNotClause,
-      should: [
-        {
-          bool: {
-            must: [
-              { match: { serviceType: 'Sexual health information and support' } },
-              { match: { venueType: 'Community' } },
-            ],
-          },
-        },
-        {
-          bool: {
-            minimum_should_match: 1,
-            must: { match: { serviceType: 'Chlamydia screening under 25s' } },
-            should: [
-              { match: { venueType: 'Clinic' } },
-              { match: { venueType: 'Community' } },
-            ],
-          },
-        },
-      ],
-    };
-  }
-  return undefined;
+queryFilters[constants.searchTypes.kitsOver25] = {
+  minimum_should_match: 1,
```

Comment:
Given there is a single must match option in the should clause, is this `minimum_should_match` required? Or could this filter be rewritten to just the `must` part e.g.
```js
must: [
  { match: { serviceType: 'SRV0531' } },
],
```

Reply:
Good suggestion, thanks, I've greatly simplified the query.
There's a few things I'd like to see addressed in another PR, such as restructuring a few related files into logical folders (rather than having them in utils), and the getFilteredOutQueryTerms test is very brittle as it's mostly repeating definitions in the source code.